configs/projects/agent-runtime-5.5.x.rb

```diff
@@ -5,13 +5,11 @@
   proj.setting :rubygem_net_ssh, '4.1.0'
   proj.setting :rubygem_semantic_puppet_version, '0.1.2'
 
-  platform = proj.get_platform
-
   # This modification to the platform definition is for compatibility with
-  # pre-puppet6 puppet-agent packaging. Fedora platforms for puppet5 must have
-  # have an `f` prefix before their version numbers and do not use an explicit
-  # .dist string.
-  if platform.name =~ /^fedora-([\d]+)/
+  # pre-puppet6 puppet-agent packaging. Fedora 26 and 27 for puppet-agent 5.5.x
+  # must have an `f` prefix before their version numbers and do not use an
+  # explicit .dist string.
+  if platform.name =~ /^fedora-2(6|7)/
```

Comment:
Dont we need 28 if its for 5.5.x?

Reply:
Or is that where we drop the f?

Reply:
This change removes the workaround for fedora 28 -- the fedora 26 and 27 platforms will still have an `f` appended dynamically during builds because of this chunk of code, but we won't be doing that for fedora 28

Reply:
so I was just talking with @branan - I think if you update all your fedora platform configs to not have the f, the only difference should be the dist tag, since RE has manually been moving things into non-f directories, e.g. http://yum.puppetlabs.com/puppet5/fedora/27/x86_64/